crypto/fipsmodule/CMakeLists.txt

```diff
@@ -152,9 +159,12 @@ function(cpreprocess dest src)
     set(TARGET "--target=${CMAKE_ASM_COMPILER_TARGET}")
   endif()
 
+  # CMAKE inserts "\" before whitespaces in arguments, which the command line does not recognize when using
+  # |add_custom_command|. Replacing the whitespaces with ";" fixes this.
+  string(REGEX REPLACE "[ ]+" ";" CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS}")
```

Comment:
I think this may have something to do with Android/NDK. Does CMAKE not insert `\` on other platforms and then break other builds?

Reply:
I could move this under `if(ANDROID)`. I'll move this along with the other change you suggested.